app/serializers/base_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class BaseSerializer
+  include ActiveModel::Serialization
+
+  def to_hash
+    self.serializable_hash.compact
```

Comment:
I believe `self` is redundant here. WDYT?

Reply:
I like the redundancy because when we read it, we don't think that `serializable_hash` is a variable. It is clear that is a method from, in this case, `self.` :eyes: